### ISSUE TITLE
Supporting self-signed certificates

### ISSUE THREAD

**Issue by aral** [libgit2/objective-git#528]
I’m using a self-signed certificate for the development environment (local testing). I see that the option to skip the certificate check was removed in d2bdd34a29e34e0127622edb19ed1999bbc0ec2f — has anything changed since then? How do we support skipping the cert check (ala `git config --global http.sslVerify false` via CLI) in Objective-Git.

(“We need to re-enable it” is a valid response but I didn’t want to do that without having a chat about it.) :)

**Comment by joshaber** [comment]
Yeah, it looks like we just haven't added support back in since we removed it. Pull requests welcome :smile:

**Comment by phatblat** [comment]
#505 has some details on this
